components/arena/python/arena/_arena_distributed_tf_op.py

```diff
@@ -29,7 +29,7 @@ def estimator_op(name, image, command,
                       evaluator=False, evaluator_cpu_limit=0, evaluator_memory_limit=0, 
                       env=[], data=[], sync_source=None,
                       metrics=['Train-accuracy:PERCENTAGE'],
-                      arena_image='cheyang/arena_launcher:v0.5',
+                      arena_image='cheyang/arena_launcher:v0.6',
```

Comment:
Should these images be moved to a more neutral docker registry name?

Reply:
I agree with your suggestions. I'd like to move it to a neutral docker registry name. I will keep it in mind and try to find. Thanks.